actionpack/lib/action_dispatch/testing/assertions/response.rb

```diff
@@ -79,7 +79,12 @@ def normalize_argument_to_redirection(fragment)
         def generate_response_message(expected, actual = @response.response_code)
           "Expected response to be a <#{code_with_name(expected)}>,"\
           " but was a <#{code_with_name(actual)}>"
-          .concat location_if_redirected
+            .concat(location_if_redirected).concat(response_body_if_short)
+        end
+
+        def response_body_if_short
+          return "" if @response.body.size > 200
```

Comment:
Here is how my terminal looks when outputting 1037 chars response:

![screen shot 2016-09-14 at 11 11 04](https://cloud.githubusercontent.com/assets/522155/18517904/faf67f2a-7a6b-11e6-80e3-713184a848c7.png)

It's still a bit too long. What about 500?